### DISCUSSION TITLE
LFG Random Bots

### DISCUSSION THREAD

**Discussion by Conan1996-2** [trickerer/Trinity-Bots#681] [Ideas]
I know you have a lot on your plate and are pretty busy, but can you keep the idea of bots randomly joining a LFG in the proper slot?

Example, if My friend (a tank) and I (a dips) join a LFG, is it possible a Healer and two DPS join as well, randomly through the LFG?

**Comment by trickerer** [comment]
Well I'm not a fan of that, I mean they can follow you but you won't be able to command them.
Or, if instead of generated bots we temporarily utilize static bots they will need their generated gear while they might have that already.
